src/shared/extensionUtilities.ts

```diff
@@ -17,41 +16,29 @@ import { getLogger } from './logger'
 const localize = nls.loadMessageBundle()
 
 export class ExtensionUtilities {
-    public static getLibrariesForHtml(names: string[]): ScriptResource[] {
+    public static getLibrariesForHtml(names: string[], webview: vscode.Webview): vscode.Uri[] {
```

Comment:
I'm fine with this file's changes, but what do you think if the methods returned `string[]` and the caller was responsible for mapping that to uris? We wouldn't need to pass webview through all these places.

Reply:
I'm fine with either. I kinda like this way because it's abstracted from the user, and in theory they should already have a webview available at this point.